src/custom/pages/Profile/LockedGnoVesting/hooks.ts

```diff
@@ -44,17 +33,22 @@ export const useAllocation = (): CurrencyAmount<Token> => {
           setAllocation(CurrencyAmount.fromRawAmount(COW, claim?.amount ?? 0))
         }
       })
+    } else {
+      setAllocation(initial)
     }
     return () => {
       canceled = true
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [chainId, account])
 
   return allocation
 }
 
 const START_TIME = 1644584715000
 const DURATION = 126144000000
+// These values match the vesting contract configuration (see contract's `cliffTime` and `duration` fields).
```

Comment:
nit: move this above the const

Reply:
Or even add one per const, and for the duration, nice to write how much time is that